modules/puppet_proxy/puppet_config.rb

```diff
@@ -1,4 +1,4 @@
-require 'augeas'
+require 'augeas' if RUBY_PLATFORM =~ /ruby/
```

Comment:
right you are; using the correct bundle install command does fix this. I revert all this back